bin/phpunit-docker.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+WP_VERSION=${1-latest}
+
+MYSQL_ROOT_PASSWORD='wordpress'
+docker network create tests
+db=$(docker run -p 3306:3306 --network tests --name db -e MYSQL_ROOT_PASSWORD=$MYSQL_ROOT_PASSWORD -d mariadb)
```

Comment:
```suggestion
db=$(docker run --network tests --name db -e MYSQL_ROOT_PASSWORD=$MYSQL_ROOT_PASSWORD -d mariadb)
```
I missed removing the host port mapping.